plugins/generator-1.21.1/neoforge-1.21.1/templates/block/block.java.ftl

```diff
@@ -664,6 +664,18 @@ public class ${name}Block extends
 	    </#if>
 	</#if>
 
+	<#if data.sensitiveToVibration && data.hasInventory>
+	@Override public <T extends BlockEntity> BlockEntityTicker<T> getTicker(Level level, BlockState blockstate, BlockEntityType<T> blockEntityType) {
+		if (!level.isClientSide && blockEntityType == ${JavaModName}BlockEntities.${data.getModElement().getRegistryNameUpper()}.get()) {

```

Comment:
I believe this is the last comment on this PR :)

Rework those to `${REGISTRYNAME}` to adapt new system of https://github.com/MCreator/MCreator/pull/5393

This applies to both MC versions